### PR TITLE
Feeture/sqone 442/panels to blocks

### PR DESCRIPTION
**Pull request by dpellenwood**
## What does this do/fix?
* Renames & moves all the "panels" code in FF to "blocks".
* Removes some old/unused "panel" templates.

[SQONE-442]

## Tests

Does this have tests?

- [ ] Yes
- [x] No, this doesn't need tests because...
- [ ] No, I need help figuring out how to write the tests.

[SQONE-442]: https://moderntribe.atlassian.net/browse/SQONE-442

### CODE REVIEW

File: wp-content/plugins/core/src/Blocks/Types/Hero/Controller.php [hunk 1]
```diff
@@ -24,14 +24,14 @@ public function render( $attributes, $content, $block_type ) {
...
 			Container::MEDIA             => $this->get_media(),
 			Container::CONTENT           => $this->get_content(),
-			Container::CONTAINER_CLASSES => [ 'hero__container', 'l-container' ],
-			Container::MEDIA_CLASSES     => [ 'hero__media' ],
-			Container::CONTENT_CLASSES   => [ 'hero__content' ],
-			Container::CLASSES           => [ 'c-panel', 'c-panel--hero', 'c-panel--full-bleed' ],
+			Container::CONTAINER_CLASSES => [ 'b-hero__container', 'l-container' ],
+			Container::MEDIA_CLASSES     => [ 'b-hero__media' ],
+			Container::CONTENT_CLASSES   => [ 'b-hero__content' ],
+			Container::CLASSES           => [ 'c-block', 'c-panel--full-bleed', 'b-hero' ],
```
Comment (jamiepaul) [inline wp-content/plugins/core/src/Blocks/Types/Hero/Controller.php:30]: `c-panel--full-bleed` should be `c-block--full-bleed`

File: wp-content/plugins/core/src/Blocks/Types/Hero/Hero.php [hunk 2]
```diff
@@ -25,8 +25,8 @@ class Hero extends Block_Type_Config {
 	public function build(): Block_Type_Interface {
 		return $this->factory->block( self::NAME )
 			->set_label( __( 'Hero', 'tribe' ) )
-			->add_class( 'c-panel c-panel--hero c-panel--full-bleed' )
-			->add_data_source( 'className-c-panel', self::LAYOUT )
+			->add_class( 'c-block c-panel--full-bleed b-hero' )
```
Comment (jamiepaul) [inline wp-content/plugins/core/src/Blocks/Types/Hero/Hero.php:28]: `c-panel--full-bleed` should be `c-block--full-bleed`

File: wp-content/plugins/core/src/Blocks/Types/Interstitial/Controller.php [hunk 3]
```diff
@@ -24,14 +24,14 @@ public function render( $attributes, $content, $block_type ) {
...
 			Container::MEDIA             => $this->get_media(),
 			Container::CONTENT           => $this->get_content(),
-			Container::CONTAINER_CLASSES => [ 'interstitial__container', 'l-container' ],
-			Container::MEDIA_CLASSES     => [ 'interstitial__media' ],
-			Container::CONTENT_CLASSES   => [ 'interstitial__content' ],
-			Container::CLASSES           => [ 'c-panel', 'c-panel--interstitial', 'c-panel--full-bleed' ],
+			Container::CONTAINER_CLASSES => [ 'b-interstitial__container', 'l-container' ],
+			Container::MEDIA_CLASSES     => [ 'b-interstitial__media' ],
+			Container::CONTENT_CLASSES   => [ 'b-interstitial__content' ],
+			Container::CLASSES           => [ 'c-block', 'c-panel--full-bleed', 'b-interstitial' ],
```
Comment (jamiepaul) [inline wp-content/plugins/core/src/Blocks/Types/Interstitial/Controller.php:30]: `c-panel--full-bleed` should be `c-block--full-bleed`

File: wp-content/plugins/core/src/Blocks/Types/Interstitial/Interstitial.php [hunk 4]
```diff
@@ -22,8 +22,8 @@ class Interstitial extends Block_Type_Config {
 	public function build(): Block_Type_Interface {
 		return $this->factory->block( self::NAME )
 			->set_label( __( 'Interstitial', 'tribe' ) )
-			->add_class( 'c-panel c-panel--interstitial c-panel--full-bleed' )
-			->add_data_source( 'className-c-panel', self::LAYOUT )
+			->add_class( 'c-block c-panel--full-bleed b-interstitial' )
```
Comment (jamiepaul) [inline wp-content/plugins/core/src/Blocks/Types/Interstitial/Interstitial.php:25]: `c-panel--full-bleed` should be `c-block--full-bleed`

**Comment by dpellenwood** [comment]
Great catches, @jamiepaul. Those should also be updated now. Thanks!

**Pull request merged**